examples/sqlalchemy.py

```diff
@@ -8,7 +8,15 @@
 catalog = os.getenv("DATABRICKS_CATALOG")
 schema = os.getenv("DATABRICKS_SCHEMA")
 
-engine = create_engine(f"databricks+thrift://token:{access_token}@{host}?http_path={http_path}&catalog={catalog}&schema={schema}")
+
+# Extra arguments are passed untouched to the driver
+# See thrift_backend.py for complete list
+extra_connect_args = {
+    "_tls_verify_hostname": True,
+    "_user_agent_entry": "Example Entry"
+}
+
+engine = create_engine(f"databricks+thrift://token:{access_token}@{host}?http_path={http_path}&catalog={catalog}&schema={schema}", connect_args=more_databricks_connect_args)
```

Comment:
does this example still works? I think you changed the path to driver name instead of thrift

Reply:
This is fixed in the latest commit (I forgot to push to this PR), whoops.